user_guide/src/howcani/combing_dataframes/concatenating.md

```diff
@@ -0,0 +1,31 @@
+# Concatenation
+
+There are a number of ways to concatenate data from separate DataFrames:
+
+- two dataframes with **the same columns** can be **vertically** concatenated to make a **longer** dataframe
+- two dataframes with the **same number of rows** and **non-overlapping columns** can be **horizontally** concatenated to make a **wider** dataframe
+- two dataframes with **different numbers of rows and columns** can be **diagonally** concatenated to make a dataframe which might be longer and/ or wider. Where column names overlap values will be vertically concatenated. Where column names do not overlap new rows and columns will be added. Missing values will be set as `null`
+
+## Vertical concatenation - getting longer
+
+In a vertical concatenation you combine all of the rows from a list of `DataFrames` into a single longer `DataFrame`.
+
+Vertical concatenation fails when the dataframes do not have the same columns.
+
+## Horizontal concatenation - getting wider
+
+In a horizontal concatenation you combine all of the columns from a list of `DataFrames` into a single wider `DataFrame`.
+
+Horizontal concatenation fails when dataframes have overlapping columns.
+
+## Diagonal concatenation - getting longer, wider and `null`ier
+
+In a diagonal concatenation you combine all of the row and columns from a list of `DataFrames` into a single longer and/or wider `DataFrame`.
+
+Diagonal concatenation generates nulls when the column names do not overlap.
```

Comment:
Maybe add another section that if the `DataFrame`'s do not match in width or height, we can join on a key that have similar semantics?

Reply:
I see that you have join section down the road. :+1:  Maybe only mention that we join we the shapes don't match and have a semantic similar key.